appserver/orb/orb-iiop/src/main/java/org/glassfish/enterprise/iiop/impl/IIOPSSLSocketFactory.java

```diff
@@ -240,43 +204,13 @@ private SSLInfo getDefaultSslInfo() throws Exception {
      * server side to create a SSLContext
      * it is called once for each serveralias and once for each clientalias
      */
-    private SSLInfo init(String alias,
-            boolean ssl2Enabled, String ssl2Ciphers,
-            boolean ssl3Enabled, String ssl3TlsCiphers,
-            boolean tlsEnabled, boolean tlsEnabled11,
-            boolean tlsEnabled12, boolean tlsEnabled13) throws Exception {
+    private SSLInfo init(String alias, boolean tlsEnabled12, boolean tlsEnabled13) throws Exception {
 
         String protocol;
         if (tlsEnabled13) {
             protocol = TLS13;
-        } else if (tlsEnabled12) {
+        } else {
```

Comment:
I wonder if this should default to 1.3, 1.2, or throw an error?
Need to debug with a reproducer to understand this and why it would originally default to SSL even though that wasn't an option.